src/Sensors/ExceptionSensor.php

```diff
@@ -102,7 +102,12 @@ private function wasManuallyReported(Throwable $e): bool
      */
     private function serializeTrace(Throwable $e): string
     {
-        $trace = [];
+        $trace = [
+            [
+                'file' => $this->location->normalizeFile($e->getFile()).':'.$e->getLine(),
+                'source' => '',
+            ],
+        ];
```

Comment:
This is similar to Symfony's approach: 

https://github.com/symfony/error-handler/blob/0b31a944fcd8759ae294da4d2808cbc53aebd0c3/Exception/FlattenException.php#L304-L339.